mora/service/employee.py

```diff
@@ -806,7 +822,7 @@ def terminate_employee(employee_uuid):
     """
     date = common.get_valid_from(flask.request.get_json())
 
-    c = lora.Connector(effective_date=date)
+    c = lora.Connector()
```

Comment:
Is this safe?

Reply:
Nope. Should probably have a test to ensure this.